src/renderer/components/Experiment/Train/TrainingModalLoRA.tsx

```diff
@@ -66,7 +76,60 @@ export default function TrainingModalLoRA({ open, onClose, experimentInfo }) {
     ),
     fetcher
   );
+  const {
+    data: templateData,
+    error: templateError,
+    isLoading: templateIsLoading,
+  } = useSWR(
+    template_id
+      ? chatAPI.Endpoints.Jobs.GetTrainingTemplate(template_id)
+      : null,
+    fetcher
+  );
 
+  async function updateTrainingTemplate(
+    template_id: string,
+    name: string,
+    description: string,
+    type: string,
+    config: string
+  ) {
+    const configBody = {
+      config: config,
+    };
+    const response = await fetch(
+      chatAPI.Endpoints.Jobs.UpdateTrainingTemplate(
+        template_id,
+        name,
+        description,
+        type
+      ),
+      {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+          accept: 'application/json',
+        },
+        body: JSON.stringify(configBody),
+      }
+    );
+    const result = await response.json();
+    return result;
+  }
+  useEffect(() => {
```

Comment:
I haven't figured out why but this isn't updating correclty in the UI even though the update is working in the background